pkg/urbit/jets/tree.c

```diff
@@ -1635,28 +1649,46 @@ static c3_c* _141_two__in_ha[] = {
     "2cc9f005fde5314e9ad545286493a8c81b5c3b775d645ad82954f405d9414a32",
     0
   };
+
+  static u3j_harm _141_two__by_all_a[] = {{".2", u3wdb_all, c3y}, {}};
+  static u3j_harm _141_two__by_any_a[] = {{".2", u3wdb_any, c3y}, {}};
+  static u3j_harm _141_two__by_apt_a[] = {{".2", u3wdb_apt, c3y}, {}};
+  static u3j_harm _141_two__by_key_a[] = {{".2", u3wdb_key, c3y}, {}};
+  static u3j_harm _141_two__by_rep_a[] = {{".2", u3wdb_rep, c3y}, {}};
+  static u3j_harm _141_two__by_run_a[] = {{".2", u3wdb_run, c3y}, {}};
+  static u3j_harm _141_two__by_urn_a[] = {{".2", u3wdb_urn, c3y}, {}};
+  static u3j_harm _141_two__by_wyt_a[] = {{".2", u3wdb_wyt, c3y}, {}};
+
   static u3j_harm _141_two__by_tap_a[] = {{".2", u3wdb_tap, c3y}, {}};
   static c3_c* _141_two__by_tap_ha[] = {
     "7dde59e2bd7684e785ce9787bc394571bd1216d7a62398c703447fc951c6b352",
     0
   };
 
   // https://github.com/urbit/urbit/issues/328
-  // static u3j_harm _141_two__by_uni_a[] = {{".2", u3wdb_uni, c3y}, {}};
-  // static c3_c* _141_two__by_uni_ha[] = {0};
+  static u3j_harm _141_two__by_uni_a[] = {{".2", u3wdb_uni, c3y}, {}};
+  static c3_c* _141_two__by_uni_ha[] = {0};
 
 static u3j_core _141_two__by_d[] =
-  { { "bif", 7, _141_two__by_bif_a, 0, _141_two__by_bif_ha },
+  { { "all", 7, _141_two__by_all_a },
+    { "any", 7, _141_two__by_any_a },
+    { "apt", 7, _141_two__by_apt_a },
+    { "bif", 7, _141_two__by_bif_a, 0, _141_two__by_bif_ha },
```

Comment:
I kinda hate to ask, but it would be better to maintain consistency in these declarations and stub out the hash vectors for all the new arms here. I'll probably go through and fill updated hashes again at some point. Until we've replaced hashboard with the new jetpack, we should fully maintain this file as is.